db/schema.up.sql

```diff
@@ -2,13 +2,17 @@
 
 DROP TABLE IF EXISTS comments;
 CREATE TABLE comments (
+  id SERIAL,
   song_id INT,
   user_id INT,
   float_id NUMERIC (11,3),
   created_at INT,
-  comment TEXT 
+  comment TEXT,
+  PRIMARY KEY (id, song_id) 
 ) PARTITION BY RANGE (song_id);
 
+CREATE INDEX songID_index ON comments USING hash (song_id);
```

Comment:
bravo! I see why your query is so fast now! You have song_id as a secondary index along with id